.github/workflows/e2e_cron.yml

```
@@ -0,0 +1,84 @@
+name: Monitor Cron e2e Tests
+on:
+  schedule:
+    - cron: '0 8 * * *'
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to run the e2e against'
+        required: false
+        default: 'stage'
+        type: choice
+        options:
+          - stage
+          - prod
+          - dev
```

Suggestion: might be good to [align these with `NODE_ENV`](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#environment-variable-load-order)?

```suggestion
          - production
          - development
```

Though assuming the latter actually points to Heroku, using `heroku` might be better, since it's easy to confuse `development` with `local`.

---

this has been updated to 'stage' and 'prod' and its meant for the test code's base url and just any other test related url needs, i can update to use 'stage' and 'production' strings if needed to match the rest of the code